spec/sidekiq/terms_of_use/sign_up_service_updater_job_spec.rb

```diff
@@ -184,6 +184,63 @@
       end
     end
 
+    context 'when agreement is changed by ToU acceptance' do
+      let(:response) { 'accepted' }
+      let(:status) { { opt_out: true, agreement_signed: false } }
+
+      before do
+        allow(service_instance).to receive(:agreements_accept)
+        allow(service_instance).to receive(:status).and_return(status)
+      end
+
+      it 'agreement_unchanged returns false' do
+        job.perform(user_account_uuid, version)
+
+        expect(MAP::SignUp::Service).to have_received(:new)
```

Comment:
I don't think we need this line, the one right  below is the better test anyways (since it tests whether the agreements accept call specifically is made)